cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -47,6 +51,15 @@ def _init_options(self, kwargs):
         if "options" not in self.options:
             self.options["options"] = {}
 
+        listeners = process_list_arg(self.options["options"].get("listeners")) or []
+
+        verbosity = int(self.options.get("perf_trace_level", 0))
+        self.robot_perf_listener = RobotPerfListener(self, verbosity)
+        listeners.append(self.robot_perf_listener)
+        if process_bool_arg(self.options.get("verbose")):
+            listeners.append(KeywordLogger)
```

Comment:
I haven't run the code so maybe it just magically works, but it seems a bit odd to have a list that includes an instance `RobotPerfListener(self, verbosity)` and a class `KeywordLogger`.

Reply:
Yeah it is a bit weird and it through me off at first but if you look at KeywordLogger you'll note that it doesn't have "self" params. So the class is used "as" just a namespace.

Reply:
I don't quite follow what you're saying. Robot will instantiate it the listener when it starts up.  I guess my point was, we might want to either pass in the name of each listener or the instance, but not one of each.

Reply:
"Robot will instantiate it the listener when it starts up."

It doesn't. It just uses the class as-is.

But sure, I can change the old code to match the new if you prefer.

412352a9d49ebf4b47e6ab2da60b874d1ee3e989

Reply:
I stand corrected. That's interesting. I've never seen a listener used this way. The only way I've ever used them was as classes that get instantiated.